File: Website/Shared/DTOs/UserNewEmailDTO.cs
namespace Domain.DTOs;

public class UserNewEmailDTO
{
    public int UserID { get; init; }
    public string NewEmail { get; init; }
    public string OldEmail { get; init; }

    public UserNewEmailDTO(int userID, string newEmail, string oldEmail)
    {
        UserID = userID;
        NewEmail = newEmail;
        OldEmail = oldEmail;
    }
}

File: Database/src/main/java/Database/DTOs/UserNewEmailDTO.java
package Database.DTOs;

public class UserNewEmailDTO
{
  private int UserID;
  private String Email;

  public UserNewEmailDTO(int userID, String email)
  {
    UserID = userID;
    Email = email;
  }

  public int getUserID()
  {
    return UserID;
  }

  public String getEmail()
  {
    return Email;
  }
}
